profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module

```diff
@@ -79,21 +79,27 @@ function nexteuropa_fields_formatters_field_formatter_settings_form($field, $ins
 
   // Switch for the different formatters.
   switch ($display['type']) {
-    case 'ne_bs_button':
+    case 'ne_button':
       $element['variation'] = [
         '#type' => 'select',
         '#title' => t('Variation'),
         '#options' => [
-          'default' => 'Default',
-          'ctn' => 'Call to action',
-          'ctn--border' => 'Call to action with border',
+          'block' => 'Default',
+          'call' => 'Call to action',
           'info' => 'Info',
-          'menu' => 'Menu',
           'primary' => 'Primary',
           'secondary' => 'Secondary',
+          'form' => 'Form',
+          'light' => 'Light',
+          'arrow-up' => 'Arrow Up',
+          'arrow-down' => 'Arrow Down',
+          'carret-up' => 'Carret Up',
+          'carret-down' => 'Carret Down',
+          'carret-left' => 'Carret Left',
+          'carret-right' => 'Carret Right',
         ],
```

Comment:
there is a typo in the values and the descriptions, Carret should be caret with one single r